recipes/apache-maven/3.6.3/conanfile.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+
+class PackageApacheMaven(ConanFile):
+    name = "apache-maven"
```

Comment:
I think that we might want to call this just maven. At least I don't know of any other popular "maven"

```suggestion
    name = "maven"
```